### PR TITLE
Send Void in response to all Batch requests

### PR DESCRIPTION
**Pull request by chbatey**
Allows application to execute batches against scassandra, just not prime or verify then.

### CODE REVIEW

**Comment by chbatey** [comment]
Hi @tolbertam if you have time would you mind reviewing this?

**Comment by tolbertam** [comment]
Sure @chbatey, i'll take a look this morning

File: server/src/main/scala/org/scassandra/server/actors/ConnectionHandler.scala [hunk 1]
```diff
@@ -147,10 +90,10 @@ class ConnectionHandler(queryHandlerFactory: (ActorRefFactory, ActorRef, CqlMess
     log.debug(s"Body length $bodyLength")
 
     if (currentData.length == bodyLength + ProtocolOneOrTwoHeaderLength) {
-      log.debug("Received exactly the whole message")
+      log.info("Received exactly the whole message")
```
Comment (tolbertam) [inline server/src/main/scala/org/scassandra/server/actors/ConnectionHandler.scala]: Is the promotion from debug -> info intentional?

Comment (chbatey) [inline server/src/main/scala/org/scassandra/server/actors/ConnectionHandler.scala]: Nope - fixing

File: server/src/test/scala/org/scassandra/server/actors/ConnectionHandlerTest.scala [hunk 2]
```diff
@@ -303,4 +320,9 @@ class ConnectionHandlerTest extends TestKit(ActorSystem("Test")) with Matchers w
     testActorRef ! Received(ByteString(startupMessage.toArray))
   }
 
+  private def clear(): Unit = {
```
Comment (tolbertam) [inline server/src/test/scala/org/scassandra/server/actors/ConnectionHandlerTest.scala]: This must just be handling any case where we get messages in between tests?  When would this happen?

**Comment by tolbertam** [comment]
looks great to me, nice and straightforward change!  :+1:

File: server/src/test/scala/org/scassandra/server/actors/ConnectionHandlerTest.scala (hunk 2 above)
Comment (chbatey) [inline server/src/test/scala/org/scassandra/server/actors/ConnectionHandlerTest.scala]: It is actually any messages not verified by previous tests. This is an issue when using the ImplicitSender trait. I asked in akka-dev and they recommend not using ImplicitSender but use a different TestProbe per test. I raised https://github.com/scassandra/scassandra-server/issues/82 so i don't forget to switch over all the tests.

Comment (tolbertam) [inline server/src/test/scala/org/scassandra/server/actors/ConnectionHandlerTest.scala]: cool, that makes sense, I like the approach you raised in the issue, :+1:

**Pull request merged**